hooks/useRewardsQueries.ts

```diff
@@ -95,9 +95,18 @@ export const useClaimRewards = ({ pool, ...options }: UseClaimRewardsArgs) => {
       const shouldBeAbleToClaimRewards = pool && client && hasPendingRewards
 
       if (shouldBeAbleToClaimRewards) {
-        const rewardsAddresses = pool.rewards_tokens?.map(
-          ({ rewards_address }) => rewards_address
-        )
+        const rewardsAddresses = pool.rewards_tokens
+          /*
+           * filter out rewards contracts that don't have pending rewards accumulated just yet.
+           * */
+          .filter((token) => {
+            const pendingRewardsForToken = pendingRewards.find(
+              ({ tokenInfo }) => token.symbol === tokenInfo.symbol
+            )
+
+            return pendingRewardsForToken.dollarValue > 0
```

Comment:
I would change this to `pendingRewardsForToken.tokenAmount > 0`

Reply:
✅